scvi/module/_vaec.py

```diff
@@ -84,7 +84,7 @@ def __init__(
             n_cat_list=[n_labels],
             n_layers=n_layers,
             n_hidden=n_hidden,
-            dropout_rate=0,
+            dropout_rate=dropout_rate,
```

Comment:
I'd see this to be zero here, see mrvdecon.py

Reply:
Same motivation.